src/modules/roc_fec/target_openfec/roc_fec/of_decoder.cpp

```diff
@@ -109,7 +109,7 @@ void OFDecoder::set(size_t index, const core::Slice<uint8_t>& buffer) {
     }
 
     if (buffer.size() == 0 || buffer.size() != payload_size_) {
-        roc_panic("of decoder: invalid payload size: size=%lu, expected=%lu",
+        roc_panic("of decoder: invalid payload size: new=%lu cur=%lu",
```

Comment:
It is more intuitive to read `cur=%lu new=%lu`, IMO.

Reply:
Agree, fixed.